embedding/cpp/include/flutter_engine.h

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_ENGINE_H_
+#define FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_ENGINE_H_
+
+#include <app_control.h>
+#include <flutter/plugin_registry.h>
+#include <flutter_tizen.h>
+
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+// The engine for flutter execution.
+class FlutterEngine : public flutter::PluginRegistry {
+ public:
+  virtual ~FlutterEngine();
```

Comment:
Please add an empty line between this line and `Create`.

Reply:
Done.